api.php

```diff
@@ -1008,6 +1008,61 @@ public function run($userID, $permissionIsExplicit) {
 	}
 }
 
+/**
+ * API entry game/getmessage
+ */
+class GetMessage extends ApiEntry {
+	public function __construct() {
+		parent::__construct('game/getmessage', 'GET', '', array('gameID','countryID','toCountryID','offset','limit'));
+	}
+	public function run($userID, $permissionIsExplicit) {
+
+		global $DB;
+		$args = $this->getArgs();
+
+		if ($args['gameID'] === null)
+			throw new RequestException('gameID is required.');
+
+		if ($args['countryID'] === null)
+			throw new RequestException('countryID is required.');
+
+		if (intval($args['limit']) > 25)
+			throw new RequestException('limit should not exceed 25');
+
+		$gameID = intval($args['gameID']);
+		$countryID = intval($args['countryID']);
+		$toCountryID = intval($args['toCountryID']);
+		$offset = intval($args['offset']);
+		$limit = intval($args['limit']);
+
+		$limit = $limit ? $limit : 25;
```

Comment:
Let's make the `25` a constant since we're also using it in an exception handler above.